pcp/columns/cmd

```diff
@@ -0,0 +1,9 @@
+# pcp-htop(1) configuration file - see pcp-htop(5)
+
+[syscmd]
+caption = myCMD
+width = -17
+description = PCP CMD description
+cmd.metric = proc.psinfo.cmd
```

Comment:
To clarify something mentioned earlier in the review, we should not have this cmd.metric syntax for columns - just use 'metric'.